### PR TITLE
Reports UI

### PR DESCRIPTION
**Pull request by mayethsc**
- Viewed Resume Report
- Login Report
- Posted Job Report

### CODE REVIEW

File: application/core/WA_Controller.php [hunk 1]
```diff
@@ -188,6 +188,15 @@ protected function loadFormContent($page = '')
         $this->load->view('forms/' . $page, $this->contentData);
     }
 
+    protected function loadReportContent($page = '')
+    {
+        if (empty($page)) {
+            $page = singular($this->currentClass);
+        }
+
+        $this->load->view('reports/' . $page, $this->contentData);
```
Comment (princeryansy) [inline application/core/WA_Controller.php]: change to $this->loadContent('reports/' . $page);

File: application/models/job/JobDetailModel.php [hunk 2]
```diff
@@ -1350,7 +1350,7 @@ private function prepareDeleted()
                 'principal_name',
                 'username',
                 'posting_date',
-                'date_deleted AS latest_update',
+                'date_deleted AS last_update',
                 '\'deleted\' as updated_by'
```
Comment (princeryansy) [inline application/models/job/JobDetailModel.php]: capitalize AS

File: application/modules/hirer/controllers/Report.php [hunk 3]
```diff
@@ -24,100 +75,273 @@ private function prepareUsers($status = 0)
...
+    private function prepareUriValues()
+    {
+        $params = $this->uri->uri_to_assoc(4, array_keys(self::URI_VALUES));
+
+        return array(
+            'f' => cleanDate(element('f', $params)),
+            't' => cleanDate(element('t', $params)),
+            'u' => cleanString(element('u', $params, '')),
+            'p' => cleanString(element('p', $params, 1)),
+
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Report.php]: remove blank line

File: application/modules/hirer/controllers/Report.php [hunk 4]
```diff
@@ -24,100 +75,273 @@ private function prepareUsers($status = 0)
...
+    private function prepareUriValues()
+    {
+        $params = $this->uri->uri_to_assoc(4, array_keys(self::URI_VALUES));
+
+        return array(
+            'f' => cleanDate(element('f', $params)),
+            't' => cleanDate(element('t', $params)),
+            'u' => cleanString(element('u', $params, '')),
+            'p' => cleanString(element('p', $params, 1)),
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Report.php]: remove comma at the end of line.

File: assets/css/table.css [hunk 5]
```diff
@@ -93,3 +93,9 @@
     border-bottom-right-radius: 5px!important;
     /*padding: 0.5rem 1rem!important;*/
 }
+
+.table td.fit,
+.table th.fit {
+    white-space: nowrap;
+    width: 1%;
+}
```
Comment (princeryansy) [inline assets/css/table.css]: add blank line at end of file.

**Comment by princeryansy** [comment]
As discussed:

- Create centralized form for reports
- Use setTitle() instead

**Comment by mayethsc** [comment]
@princeryansy , done updating requested changes.

File: application/modules/hirer/controllers/Report.php [hunk 6]
```diff
@@ -144,7 +144,7 @@ public function login($page = 1)
         $this->table->set_heading(self::LOGIN_HEADERS);
         $this->table->set_template($config);
 
-        $this->setTitle('Login Report');
+        $this->setTitle(lang('label_login_report'));
```
Comment (princeryansy) [inline application/modules/hirer/controllers/Report.php]: use title_ prefix instead, apply to similar occurrences

**Comment by princeryansy** [comment]
ACR Approved!

**Pull request merged**